projects/azure/image-mirror/image-mirror.yaml

```diff
@@ -0,0 +1,60 @@
+kind: List
+apiVersion: v1
+items:
+
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: image-mirror
+    namespace: azure
+  data:
+    openshift-azure_v3_10_quay: |
+      registry.svc.ci.openshift.org/azure/azure-plugins:sync quay.io/openshift-on-azure/sync:v3.10 quay.io/openshift-on-azure/sync:v3.10.0 quay.io/openshift-on-azure/sync:latest
+
+- apiVersion: batch/v1beta1
+  kind: CronJob
+  metadata:
+    name: "image-mirror-openshift-azure-v3.10-quay"
+    namespace: azure
+    labels:
+      app: "image-mirror-openshift-azure-v3.10-quay"
+  spec:
+    schedule: "0 * * * *"
+    concurrencyPolicy: Forbid
+    successfulJobsHistoryLimit: 1
+    failedJobsHistoryLimit: 2
+    jobTemplate:
+      spec:
+        template:
+          metadata:
+            name: "image-mirror-openshift-azure-v3.10-quay"
+            labels:
+              app: "image-mirror-openshift-azure-v3.10-quay"
+          spec:
+            restartPolicy: OnFailure
+            volumes:
+            - name: push
+              secret:
+                secretName: openshift-on-azure-openshiftci-secret  # generate from quay console
```

Comment:
Where is this coming from?

Reply:
See the comment, needs to be generated from quay

Reply:
So same as with other secrets. No makefile for creation for it?

Reply:
glad it's temporary - will be happy to see this being push-based.